unit_tests/test_version_utils.py

```diff
@@ -12,6 +12,8 @@
           '9f724fedb93b4734fcfaec1156806921ff46e956-2bdfa9f7ef592edaf15e028faf3b7f695f39ebc1'\
           '-525a0255f73d454f8f97f32b8bdd71c8dec35d3d-a6b2b2355c666b1893f702a587287da978aeec22/71/scylla.repo'
 
+TMP_RPM_URL = 'http://scratch.scylladb.com/amos/yum_repos/master/piotrj_issue5288_v3/scylla.repo'
```

Comment:
Just change the name of this variable even with the issue number

Reply:
You expect the variable to be renamed?
TMP_RPM_URL -> ISSUE_5288_RPM_URL ?